### PR TITLE
[CDAP-6464] add sort by tracker meter

### PR DESCRIPTION
**Pull request by elfenheart**
adding sort by tracker meter (ascending/descending)

Build: http://builds.cask.co/browse/CDAP-DRC4226-1
JIRA: https://issues.cask.co/browse/CDAP-6464

### CODE REVIEW

File: cdap-ui/app/features/tracker/controllers/results-ctrl.js [hunk 1]
```diff
@@ -326,6 +334,10 @@ class TrackerResultsController {
       .$promise
       .then((response) => {
         this.truthMeterMap = response;
+        angular.forEach(this.fullResults, (entity) => {
+          entity.meter = response[entity.entityTypeState][entity.name];
```
Comment (ajainarayanan) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:345]: Would there be a possibility that we don't get a truthmeter for a particular entity? Like no entry in `response`?

**Comment by ajainarayanan** [comment]
Minor comment. Otherwise 👍 LGTM

File: cdap-ui/app/features/tracker/controllers/results-ctrl.js (hunk 1 above)
Comment (elfenheart) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:345]: it should not be possible, but i'll add a safeguard anyways.

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**